src/marqo_commons/model_registry/clip_properties.py

```diff
@@ -11,64 +11,56 @@
 class ClipProperties(ModelProperties):
     vector_numeric_type: VectorNumericType = VectorNumericType.float32
     modality: List[Modality] = [Modality.image, Modality.text]
+    default_memory_size: float = 1.0
     type: str = "clip"
 
 
 def _get_clip_properties() -> Dict:
     CLIP_MODEL_PROPERTIES = {
-        'RN50': vars(ClipProperties(
+        'RN50': ClipProperties(
             name="RN50",
-            memory_size=1,
             dimensions=1024,
             notes="CLIP resnet50",
-        )),
-        'RN101': vars(ClipProperties(
+        ).to_dict(),
+        'RN101': ClipProperties(
             name="RN101",
-            memory_size=1,
             dimensions=512,
             notes="CLIP resnet101",
-        )),
-        'RN50x4': vars(ClipProperties(
+        ).to_dict(),
+        'RN50x4': ClipProperties(
             name="RN50x4",
-            memory_size=1,
             dimensions=640,
             notes="CLIP resnet50x4",
-        )),
-        'RN50x16': vars(ClipProperties(
+        ).to_dict(),
+        'RN50x16': ClipProperties(
             name="RN50x16",
-            memory_size=1,
             dimensions=768,
             notes="CLIP resnet50x16",
-        )),
-        'RN50x64': vars(ClipProperties(
+        ).to_dict(),
+        'RN50x64': ClipProperties(
             name="RN50x64",
-            memory_size=1,
             dimensions=1024,
             notes="CLIP resnet50x64",
-        )),
-        'ViT-B/32': vars(ClipProperties(
+        ).to_dict(),
+        'ViT-B/32': ClipProperties(
             name="ViT-B/32",
-            memory_size=1,
             dimensions=512,
             notes="CLIP ViT-B/32",
-        )),
-        'ViT-B/16': vars(ClipProperties(
+        ).to_dict(),
+        'ViT-B/16': ClipProperties(
             name="ViT-B/16",
-            memory_size=1,
             dimensions=512,
             notes="CLIP ViT-B/16",
-        )),
-        'ViT-L/14': vars(ClipProperties(
+        ).to_dict(),
+        'ViT-L/14': ClipProperties(
             name="ViT-L/14",
-            memory_size=1.5,
             dimensions=768,
             notes="CLIP ViT-L/14",
-        )),
-        'ViT-L/14@336px': vars(ClipProperties(
+        ).to_dict(),
+        'ViT-L/14@336px': ClipProperties(
             name="ViT-L/14@336px",
-            memory_size=1.5,
             dimensions=768,
             notes="CLIP ViT-L/14@336px",
-        )),
+        ).to_dict(),
```

Comment:
Not a need, but there's a way to remove the hassle of adding `.to_dict()` to the end of every properties object we instantiate. We can just add a loop to the end:

```
CLIP_MODEL_PROPERTIES = {key: value.to_dict() for key, value in CLIP_MODEL_PROPERTIES}
return CLIP_MODEL_PROPERTIES
```